src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+
+5. **Network Upgrade Activated**: after the network upgrade is activated, the EIP will be live on the Ethereum network. _Note: network upgrades are usually activated on testnets before being activated on the Ethereum Mainnet._
+
+This flow, while very simplified, gives an overview of the significant stages for a protocol change to be activated on Ethereum. Now, let's look at the informal factors at play during this process.
+
+## Informal Process {#informal-process}
+
+### Understanding Prior Work {#prior-work}
+
+EIP Champions should familiarise themselves with prior work and proposals before having an EIP, which can be seriously considered for deployment on the Ethereum Mainnet. This way, the EIP hopefully brings something new which hasn't been rejected before. The three main places to research this are the [EIP repository](https://github.com/ethereum/eips), [Ethereum Magicians](https://www.ethereum-magicians.org/) and [ethresear.ch](https://www.ethresear.ch/).
+
+### Working Groups {#working-groups}
+
+It is unrealistic to expect that the initial EIP will be included on the Ethereum Mainnet without change for more significant initiatives. In practice, these will require EIP Champions to work with a subset of Protocol Developers to specify, implement, test, iterate on and finalize their proposal. Historically, these working groups have involved several months (and sometimes years!) of work. Similarly, EIP Champions for such changes should involve relevant Application/Tooling Developers early in their efforts to gather end-user feedback and mitigate any deployment risks.
+
+### Community Consensus {#community-consensus}
+
+Unfortunately, there is no clear playbook on how to handle contentious proposals. Because they have no way to force people to adopt network upgrades, Protocol Developers will generally avoid including EIPs who are contentious amongst various stakeholder groups.
+
+As the champion of a contentious EIP, you should try and build consensus, address objections and document community support for your proposal. Given the size and diversity of the Ethereum community, there isn't a single metric (e.g. a coin vote) that can be used, and EIP Champions are expected to adapt to the circumstances of their proposal.
+
+Beyond the security of the Ethereum network, which contentious forks may compromise, significant weight is usually placed on what Application/Tooling Developers and Application Users value, given that their using and developing on Ethereum is what makes the ecosystem attractive for other stakeholders.
+
+<Divider />
+
+## Handling disagreements {#disagreements}
+
+Having many stakeholders with different motivations and beliefs means that disagreements are not uncommon.
```

Comment:
```suggestion
Having [many stakeholders](/community/) with different motivations and beliefs means that disagreements are not uncommon.
```

Reply:
I think this link is a bit distracting. A user would click on it expecting to see more specific stakeholder information but instead, get the less specific community page.